File: examples/dd.cpp
sim.set_num_qubits(4);
// Unsupported operation: delay
sim.gate_x(0, std::vector<int>{}, std::vector<int>{});
// Unsupported operation: delay
sim.gate_y(0, std::vector<int>{}, std::vector<int>{});
// Unsupported operation: delay
sim.gate_x(0, std::vector<int>{}, std::vector<int>{});
// Unsupported operation: delay
sim.gate_y(0, std::vector<int>{}, std::vector<int>{});
// Unsupported operation: delay
sim.gate_x(3, std::vector<int>{}, std::vector<int>{2});
sim.gate_x(2, std::vector<int>{}, std::vector<int>{1});
sim.gate_u3(0.0, 0.0, 0.0, 0.0, 3, std::vector<int>{}, std::vector<int>{});

File: examples/dd.py
from qiskit import QuantumCircuit

qc = QuantumCircuit(4)

qc.delay(1, 0)
qc.x(0)
qc.delay(1, 0)
qc.y(0)
qc.delay(1, 0)
qc.x(0)
qc.delay(1, 0)
qc.y(0)
qc.delay(1, 0)

qc.cx(2, 3)
qc.cx(1, 2)
qc.u(0.0, 0.0, 0.0, 3)
